### PR TITLE
Bookstore: Initialize with components

### PR DESCRIPTION
**Pull request by amare1990**
- Setup `Gitflow`
- Cloned to the local computer.
- Created React application inside the repository using `create-react-app .` command.
- Added linters.
- Created Components and checked preliminary functionality.
- Checked linters.

### CODE REVIEW

File: src/components/Categories.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Categories = () => (
+  <>
+    <h1> Under Construction</h1>
+  </>
+);
+
+export default Categories;
```
Comment (roshan-bajgain) [inline src/components/Categories.js]: - [ ] According to the project requirement, you should display a button with the text "Check status". Not the heading. Please implement it

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @amare1990, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️
- [ ] [Optional] When I navigate to the `categories` page then there is no option to render in the `book` component. Please make the routes visible when we are on the categories page as well. And it makes the UX design better and users can navigate to `book` and `categories` easily.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Categories.js (hunk 1 above)
Comment (amare1990) [reply src/components/Categories.js]: Hi @roshan-bajgain. Thanks for your quite clear feedback. Now the problem seems to be fixed.

**Review by roshan-bajgain — APPROVED** [review]
Hi @amare1990, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**